ui/components/app/snaps/snap-ui-renderer/index.scss

```diff
@@ -84,4 +84,12 @@
       max-width: $width-screen-lg-min;
     }
   }
+
+  &__form {
```

Comment:
Is this only a problem in forms? Wouldn't this apply to all boxes?

cc @GuillaumeRx

Reply:
Makes sense, let's consider this narrow case for now

Reply:
Fine with me to start with a limited scope. We should ensure that the the behavior is also the same outside of forms later on, we allow Fields outside of forms IIRC